FoxHound.Client/src/Content.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
Consider moving the entire Content.tsx file into the Navigation folder since it relates to nav.